src/Compilers/Core/Portable/Compilation/IOperation.cs

```diff
@@ -104,6 +104,10 @@ public enum OperationKind
         // Newly added
 
         ConditionalAccessExpression,
-        IncrementExpression
+        IncrementExpression,
+
+        Argument,
```

Comment:
How are you planning to represent omitted arguments for parameters that do not have defaults? This can happen in error scenarios.